README.md

```diff
@@ -1,2 +1,2 @@
-# example-arm
-ros2 example application specific for arm architecture
+# Build arm examples
+Documentation is in the wiki: https://github.com/ros2-for-arm/ros2/wiki/Secure-messaging-example
```

Comment:
Is the wiki in "ros2" not "example"?

Reply:
The only wiki will be the ros2-for-arm/ros2 one. We want to centralise all the information at the same place. All the readme will point to this wiki.